packages/auth/src/mfa/assertions/totp.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { TotpSecret } from '../../platform_browser/mfa/assertions/totp';
+import { TotpMultiFactorAssertion } from '../../model/public_types';
+import { FactorId } from '../../model/enum_maps';
```

Comment:
We shouldn't use this import in the SDK. The objects in `enum_maps.ts` are a convenience for external developers, but they can't be tree shaken. Instead, import the `FactorId` `const enum` from `public_types` and use that instead. The compiler will convert `FactorId.TOTP` to `'totp'`